Sources/XcodeProj/Objects/SwiftPackage/XCSwiftPackageProductDependency.swift

```diff
@@ -18,20 +18,28 @@ public class XCSwiftPackageProductDependency: PBXContainerItem, PlistSerializabl
         }
     }
 
+    /// Is it a Plugin.
+    var isPlugin: Bool
+
     // MARK: - Init
 
     public init(productName: String,
-                package: XCRemoteSwiftPackageReference? = nil) {
+                package: XCRemoteSwiftPackageReference? = nil,
+                isPlugin: Bool = false) {
         self.productName = productName
         packageReference = package?.reference
+        self.isPlugin = isPlugin
         super.init()
     }
 
     public required init(from decoder: Decoder) throws {
         let objects = decoder.context.objects
         let repository = decoder.context.objectReferenceRepository
         let container = try decoder.container(keyedBy: CodingKeys.self)
-        productName = try container.decode(String.self, forKey: .productName)
+        let rawProductName = try container.decode(String.self, forKey: .productName)
+        productName = rawProductName.replacingOccurrences(of: "plugin:", with: "")
+        isPlugin = productName != rawProductName
```

Comment:
perhaps we can be a bit more explicit here?

```suggestion
        let pluginPrefix = "plugin:"
        if rawProductName.hasPrefix(pluginPrefix) {
            productName = String(rawProductName.dropFirst(pluginPrefix.count))
            isPlugin = true
        } else {
            productName = rawProductName
            isPlugin = false
        }
```

Reply:
Looks more readable 👍